docs/mp/beanvalidation/02_general_beanvalidation.adoc

```diff
@@ -67,5 +67,7 @@ public class GreetingProvider {
 }
 ----
 
+NOTE: `beans.xml` is required to identify beans and for bean validation to work properly.
+
 More information about validations supported is available can be found in {bean-validation-spec-url}[Jakarta Bean
```

Comment:
For more information about the supported validations, see {bean-validation-spec-url}[Jakarta Bean Validation specification].